utkdigitalinitiatives/utk-islandora7-drupal
# Upgrade video-js, packaged banner updates, mark.js.

### What is this?

The commit packages the _video-js_ update (7.6.4 => 7.9.2) completed for the temporary RFTA collection on dlwork, any banner updates made since June, and changes to the mark.js file.

The goal of this is to sync the file structures of dlwork and digital to prep for a dev -> staging -> production workflow.

### How do I test this?

- check that **themes/banners/*** directory contents match across dlwork and digital.
- check that **themes/sis/js/mark.js** is the same file on both dlwork and digital
- video.js updates will be testing further in staging (Porter) before being deployed to production (Digital)
